react/features/base/tracks/middleware.web.ts

```diff
@@ -62,7 +71,32 @@ MiddlewareRegistry.register(store => next => action => {
 
     case TRACK_REMOVED: {
         _removeNoDataFromSourceNotification(store, action.track);
-        break;
+
+        const result = next(action);
```

Comment:
Do we need to log it here since with ssrc-rewriting, tracks will never be removed.

Reply:
It could be useful for the case where the ssrc rewriting is not enabled.

Reply:
Even without ssrc rewriting, remote tracks are removed only when participants leave. Do you think track states will be helpful in that case. I am just worried about all the unnecessary logging.